kaldb/src/main/java/com/slack/kaldb/clusterManager/SnapshotDeletionService.java

```diff
@@ -187,6 +187,7 @@ protected int deleteExpiredSnapshotsWithoutReplicas() {
                               // the
                               // metadata and try again on the next run.
                               URI snapshotUri = URI.create(snapshotMetadata.snapshotPath);
+                              LOG.info("Starting delete of snapshot {}, snapshot URI of {}", snapshotMetadata.name, snapshotMetadata.snapshotPath);
```

Comment:
I think it may be better to log the entire snapshot metadata here to help with other debugging scenarios.

Reply:
Done. There should be relatively infrequent amounts of snapshots being deleted (1k / week) that this isn't ever an issue if we decide to permanently keep it.